src/domain/synthetics/positions/utils.ts

```diff
@@ -393,6 +394,10 @@ export function getMinCollateralFactorForPosition(position: PositionInfo, openIn
   return minCollateralFactor;
 }
 
+export function getIsPositionInfoLoaded(pos?: PositionInfo): pos is PositionInfoLoaded {
```

Comment:
Strange signature of typeguard, even not sure why does it work as PositionInfoLoaded is supertype of PositionInfo. Maybe it's better to explicitly say that pos `PositionInfo | PositionInfoLoaded`